connection.cc

```diff
@@ -57,9 +57,10 @@ bool Connection::handle_read(const boost::system::error_code& error,
     // Default request handler
     RequestHandler* request_handler = &echo_request_handler; 
 
-    if (static_map->find(request.handler_path) != static_map->end()) {
+    if (static_map_->count(request.handler_path) > 0) {
       request_handler = &static_request_handler;
-    } else if (echo_map->find(request.handler_path) == echo_map->end()) {
+      request.base_path = static_map_->at(request.handler_path);
+    } else if (echo_map_->count(request.handler_path) == 0) {
       // Directory for request is not found in either
       // For the sake of this implementation, continue with EchoRequestHandler
       std::cerr << "Couldn't find request directory in echo and static map" << std::endl;
```

Comment:
can you update the error statement to be "Couldn't find request handler_path in echo and static map" ?